flake8_quotes/__init__.py

```diff
@@ -12,31 +12,53 @@ class QuoteChecker(object):
     QUOTES = {
         # When user wants only single quotes
         '\'': {
-            'good_single': '\'',
-            'good_multiline': '\'\'\'',
-            'bad_single': '"',
-            'bad_multiline': '"""',
+            'good': '\'',
+            'bad': '"',
         },
+
+        # When user wants only double quotes
         '"': {
-            'good_single': '"',
-            'good_multiline': '"""',
-            'bad_single': '\'',
-            'bad_multiline': '\'\'\'',
+            'good': '"',
+            'bad': '\'',
         },
     }
 
+    MULTILINE_QUOTES = {
```

Comment:
I just used dicts as that's what was being used for `QUOTES`, I find it makes it easier to read and there's no real advantage of using a list either.